tests/small/LandIce/FO_GIS/input_fo_humboldt_frosch_fluxdiv.yaml

```diff
@@ -105,7 +115,7 @@ ANONYMOUS:
     Exodus Output File Name: output/humboldt.exo
     Element Shape: Wedge
     Use Serial Mesh: false
-    Columnwise Ordering: true
+    Columnwise Ordering: false
```

Comment:
What's the reasoning behind this change?

Reply:
The new response works only for contiguous maps (it would error out otherwise). When the columnwise ordering is true, the basal map is not contiguous.